$MirOS: ports/emulators/qemu/snapshot/patches/patch-target-mips_op_helper_c,v 1.1 2008/05/07 15:53:13 tg Exp $
--- target-mips/op_helper.c.orig	Wed Sep 23 19:00:50 2009
+++ target-mips/op_helper.c	Sat Oct 17 16:36:24 2009
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>.
  */
+#include <stdio.h>
 #include <stdlib.h>
 #include "exec.h"
 
